sdk/communication/communication-short-codes/swagger/README.md

```diff
@@ -12,7 +12,7 @@ generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
 tag: package-shortcode-2021-10-25-preview
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/ca0335b44b4eca2c5b5673ee2c58a87e524b669f/specification/communication/data-plane/ShortCodes/readme.md
```

Comment:
This seems nonstandard. Why aren't you referencing a swagger from the `azure-rest-api-specs` repo? Has the API been reviewed by the Azure Board?

Reply:
The API went private mode, as suggested by the Azure Board, so we had to remove the swagger from the spec repo. The package, however, still needs to be released, so that's why we're using a local file instead. Does that make sense for you?

Reply:
I wasn't aware of that you're in the private mode and the swagger was removed. Thanks for explaining.